<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Winwork</class>
 <widget class="QMainWindow" name="Winwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listContacts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="butSay">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Say</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listDialog">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>31</y>
      <width>531</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans Mono</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMessage">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
